ci/common/test.sh

```diff
@@ -131,8 +132,7 @@ check_runtime_files() {(
   local message="$1" ; shift
   local tst="$1" ; shift
 
-  cd runtime
-  for file in $(git ls-files "$@") ; do
+  for file in $(git -C runtime ls-files "$@") ; do
     # Check that test is not trying to work with files with spaces/etc
     # Prefer failing the build over using more robust construct because files
     # with IFS are not welcome.
```

Comment:
I don't think `file` has the full/absolute path, so the `test -e ...` fails.

```
E /tests/install_nvim|runtime-install :: It appears that autoload/RstFold.vim is only a part of the file name
E /tests/install_nvim|runtime-install :: It appears that autoload/ada.vim is only a part of the file name
E /tests/install_nvim|runtime-install :: It appears that autoload/adacomplete.vim is only a part of the file name
E /tests/install_nvim|runtime-install :: It appears that autoload/ccomplete.vim is only a part of the file name
E /tests/install_nvim|runtime-install :: It appears that autoload/clojurecomplete.vim is only a part of the file name
```

Rather than work around this, it's better just to keep the `cd runtime`. Using subshells is very common in various shell scripts throughout the project.

Reply:
Oh, I've missed that it was using a subshell in the function now..

Will remove this commit here then.  Changing to a WIP for minor things.